image/image_linux.go

```diff
@@ -272,302 +273,229 @@ func makeLabel(now time.Time) string {
 	return now.UTC().Format("20060102")
 }
 
-func selectAssertionMaxFormats(tsto *tooling.ToolingStore, sysSn *seedwriter.SeedSnap) error {
-	if sysSn == nil {
-		// nothing to do
-		return nil
+type imageSeeder struct {
+	classic        bool
+	prepareDir     string
+	wideCohortKey  string
+	revisions      map[string]snap.Revision
+	customizations *Customizations
+
+	hasModes    bool
+	rootDir     string
+	bootRootDir string
+	seedDir     string
+	label       string
+
+	model *asserts.Model
```

Comment:
I meant first of the whole struct, so before classic bool

Reply:
Fixed!